spark-operator/build.gradle

```diff
@@ -0,0 +1,90 @@
+apply plugin: 'com.github.johnrengelman.shadow'
+
+buildscript {
+    repositories {
```

Comment:
It seems to be a mistake because we decide to use 2-space indentation in Gradle file.

https://github.com/apache/spark-kubernetes-operator/blob/d7734bbc4413163cf60fe67e23c541929a9a37a8/build.gradle#L1-L2

https://github.com/apache/spark-kubernetes-operator/blob/d7734bbc4413163cf60fe67e23c541929a9a37a8/spark-operator-api/build.gradle#L1-L2

https://github.com/apache/spark-kubernetes-operator/blob/d7734bbc4413163cf60fe67e23c541929a9a37a8/spark-submission-worker/build.gradle#L1-L2

It seems that we need to improve our CI to fail this kind of issues later.

Reply:
definitely. I would fix this with a JIRA.